code/src/components/Form.js

```diff
@@ -0,0 +1,105 @@
+import React, {useState} from 'react'
+
+import { NameInput } from 'components/NameInput'
+import { Counter } from 'components/Counter'
+import { Welcome } from 'components/Welcome'
+
+
+import HaveYouWatched from 'components/HaveYouWatched'
+import Summary from 'components/Summary'
+import WhatShip from 'components/WhatShip'
+
+
+const Form = () => {
+
+  const [step, setStep] = useState(1)
+  const [watchedStatus, setWatchedStatus] = useState('')
+  const [nameInput, setNameInput] = useState('')
+  const [whatTeam, setShip] = useState('')
+  const [value, setValue] = useState("50")
+
+
+
+  const valueUpdateInput = (event) => {
+    setValue(event.target.value)
+  }
+  
+
+  const nameUpdateInput = (event) => {
+    setNameInput(event.target.value)
+}
+
+  const stepChanger = () => {
+    setStep(step +1)
+    
+  }
+
+  const onWatchedChange = (status) => {
+      setWatchedStatus(status)
+  }
+
+  const teamUpdate = (dude) => {
+    setShip(dude)
+}
+
+
+document.addEventListener('click', e => {
+  if (e.keyCode === '13' && step < 6) {
+    setStep(step +1);
+  } else if (e.keyCode === '37' && step > 1) {
```

Comment:
I was wondering what this else if statement is doing with keyCode 37, which seems to be the arrow left button, it should go one question back? It didn't work for me.

Reply:
You're correct, I wanted a function for going backwards if pressing the left arrow key and forwards when pressing enter, but I didn't get it to work, should remove this.